frameworks/Java/tio-http-server/tio-server.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM litongjava/maven:3.8.8-jdk8u391 AS builder
+WORKDIR /app
+
+COPY pom.xml pom.xml
+RUN mvn dependency:go-offline
```

Comment:
Please add `--quiet`
Now it's creating +5k useless lines in the logs.
(progress ....)
https://github.com/TechEmpower/FrameworkBenchmarks/actions/runs/11042481015/job/30945138563#step:9:5723

Reply:
@joanhey Thanks, I've fixed. Could you please help review it? Thank you for your time!